### PR TITLE
sway-output(5): Added additional info for scaling and position wrt. scale

### PR DESCRIPTION
**Pull request by PlusMinus0**
Some more clarifications because it seems scale questions are recurring.

### CODE REVIEW

File: sway/sway-output.5.scd [hunk 1]
```diff
@@ -38,17 +38,20 @@ must be separated by one space. For example:
...
 
 		output HDMI1 scale 2
 
-		output HDMI1 pos 0 0 res 3200x1800
+		output HDMI1 pos 0 1020 res 3200x1800
 
 		output eDP1 pos 1600 0 res 1920x1080
 
-	Note that the x-pos of eDP1 is 1600 = 3200/2.
+	Note that the x-pos of eDP1 is 1600 = 3200/2 and the _bottom_ y-pos is
```
[the remark this answers is not in the document]
Comment (PlusMinus0) [reply sway/sway-output.5.scd]: Can the x-pos be top or bottom?

[the remark this answers is not in the document]
Comment (emersion) [reply sway/sway-output.5.scd]: My bad, s/top/left/

File: sway/sway-output.5.scd [hunk 2]
```diff
@@ -57,7 +60,8 @@ must be separated by one space. For example:
 	represent the contents of your windows - they will be rendered at the next
 	highest integral scale factor and downscaled. You may be better served by
 	setting an integral scale factor and adjusting the font size of your
-	applications to taste.
+	applications to taste. For scales other than 1, xwayland windows will look 
+	blurry for that screen.
```
[the remark this answers is not in the document]
Comment (PlusMinus0) [reply sway/sway-output.5.scd]: Agreed, but the blur is an obvious symptom. 

Maybe: “ HiDPI isn't supported with Xwayland clients (rendering will be blurry).”

**Comment by emersion** [comment]
Thanks!

**Pull request merged**